boards/infineon/cyw920829m2evk_02/doc/index.rst

```diff
@@ -0,0 +1,125 @@
+
+- `CYW920829M2EVK-02 Website`_
+- `CYW920829M2EVK-02 BT User Guide`_
+
+
+Kit Features:
+=============
+
+- AIROC™ CYW20829 Bluetooth® LE MCU in 56 pin QFN package
+- Arduino compatible headers for hardware expansion
+- On-board sensors - 6-axis IMU, Thermistor, Infineon analog microphone, and Infineon digital microphone
+- User switches, RGB LED and user LEDs
+- USB connector for power, programming and USB-UART bridge
+
+Kit Contents:
+=============
+
+- CYW20829 evaluation board (CYW9BTM2BASE3+CYW920829M2IPA2)
+- USB Type-A to Micro-B cable
+- Six jumper wires (five inches each)
+- Quick start guide
+
+
+Supported Features
+==================
+
+The board configuration supports the following hardware features:
+
++-----------+------------+-----------------------+
+| Interface | Controller | Driver/Component      |
++===========+============+=======================+
+| NVIC      | on-chip    | nested vectored       |
+|           |            | interrupt controller  |
++-----------+------------+-----------------------+
+| SYSTICK   | on-chip    | system clock          |
++-----------+------------+-----------------------+
+| GPIO      | on-chip    | GPIO                  |
++-----------+------------+-----------------------+
+| UART      | on-chip    | serial port-polling;  |
+|           |            | serial port-interrupt |
++-----------+------------+-----------------------+
+
+
+The default configuration can be found in the Kconfig
+
+:zephyr_file:`boards/infineon/cyw920829m2evk_02/cyw920829m2evk_02_defconfig`
+
+System Clock
+============
+
+The AIROC™ CYW20829 Bluetooth®  MCU SoC is configured to use the internal IMO+FLL as a source for
+the system clock. Other sources for the system clock are provided in the SOC, depending on your
+system requirements.
+
+Build blinking led sample
+*************************
+
+Here is an example for the :zephyr:code-sample:`blinky` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: cyw920829m2evk_02
+   :goals: build flash
+
+OpenOCD Installation
+====================
+
+To get the OpenOCD package, it is required that you
+
+1. Download and install the `ModusToolbox`_ software.
+2. After the installation, add the directory containing the OpenOCD scripts to your environment's PATH variable.
+
+
+Programming and Debugging
+*************************
+
+The CYW920829M2EVK-02 includes an onboard programmer/debugger (KitProg3) to provide debugging, flash programming, and serial communication over USB. Flash and debug commands must be pointed to the Cypress OpenOCD you downloaded above.
+
+On Windows:
+
+.. code-block:: console
+
+   west flash --openocd path/to/infineon/openocd/bin/openocd.exe
+   west debug --openocd path/to/infineon/openocd/bin/openocd.exe
+
+On Linux:
+
+.. code-block:: console
+
+   west flash --openocd path/to/infineon/openocd/bin/openocd
+   west debug --openocd path/to/infineon/openocd/bin/openocd
```

Comment:
```suggestion
.. code-block:: shell

   west flash --openocd path/to/infineon/openocd/bin/openocd.exe
   west debug --openocd path/to/infineon/openocd/bin/openocd.exe

On Linux:

.. code-block:: shell

   west flash --openocd path/to/infineon/openocd/bin/openocd
   west debug --openocd path/to/infineon/openocd/bin/openocd
```

Reply:
Thank you for the +1, I am working on a PR adding support for 20829 drivers, I will fix this in that PR.